packages/core/src/consumed-thing.ts

```diff
@@ -219,7 +219,7 @@ class InternalPropertySubscription extends InternalSubscription {
         for (let i = 0; i < forms.length; i++) {
             let score = 0;
             const form = forms[i];
-            if (form.op === operation || (Array.isArray(form.op) && form.op.includes(operation))) {
+            if (form.op === operation || (form?.op?.includes(operation) === true && Array.isArray(form.op) === true)) {
```

Comment:
I think theoretically, the `&& Array.isArray(form.op) === true` part could also be removed here. However, then the check would also return `true` for strings such as `unobservepropertyfoo` – would that be a realistic scenario we should deal with?

Reply:
Let's keep it as is..